src/lib/utils.ts

```diff
@@ -70,3 +69,11 @@ export function reminderTime({ expires: date, repeat }: Reminder): string {
 
 	return prettyDateTime;
 }
+
+export function calcNeededExp(levelExp: number, direction: string): number {
+	const xpRatio = 1.25;
```

Comment:
This probably should be a constant in config if we're going to use it in messageCount and also here